Extensions/TweenBehavior/tweentools.ts

```diff
@@ -59,6 +61,126 @@ namespace gdjs {
           : shifty.Tweenable.formulas.linear;
         return fromValue + (toValue - fromValue) * easingFunction(weighting);
       };
+
+      const getTweensMap = (runtimeScene: RuntimeScene) =>
+        runtimeScene._tweens || (runtimeScene._tweens = new Map());
+      const getShiftyScene = (runtimeScene: RuntimeScene) =>
+        runtimeScene.shiftyJsScene ||
+        (runtimeScene.shiftyJsScene = new shifty.Scene());
+
+      export const sceneTweenExists = (
+        runtimeScene: RuntimeScene,
+        id: string
+      ) => getTweensMap(runtimeScene).has(id);
+
+      export const sceneTweenIsPlaying = (
+        runtimeScene: RuntimeScene,
+        id: string
+      ) => {
+        const tweenMap = getTweensMap(runtimeScene);
+        const tween = tweenMap.get(id);
+        return tween && tween.isPlaying();
```

Comment:
Everywhere:
```suggestion
        return !!tween && tween.isPlaying();
```

Otherwise you risk returning something that is not a boolean (and risk trigger a de-opt in the JIT as the return type is not consistent)